ESP8266/ESP8266.cpp

```diff
@@ -538,32 +557,26 @@ void ESP8266::_oob_packet_hdlr()
     if (!_parser.recv(",%d,", &id)) {
         return;
     }
-    // In passive mode amount not used...
-    if (_tcp_passive
-            && _sock_i[id].open == true
-            && _sock_i[id].proto == NSAPI_TCP) {
-        if (!_parser.recv("%d\n", &amount)) {
-            MBED_ERROR(MBED_MAKE_ERROR(MBED_MODULE_DRIVER, MBED_ERROR_CODE_ENODATA), \
-                       "ESP8266::_packet_handler(): Data length missing");
+
+    if(_tcp_passive && _sock_i[id].open == true && _sock_i[id].proto == NSAPI_TCP) {
+        if (_parser.recv("%d\n", &amount)) {
+            _sock_i[id].tcp_data_avbl = amount; // Not used but stored for the sake of visibility
         }
         return;
-        // Amount required in active mode
     } else if (!_parser.recv("%d:", &amount)) {
         return;
     }
 
     pdu_len = sizeof(struct packet) + amount;
 
     if ((_heap_usage + pdu_len) > MBED_CONF_ESP8266_SOCKET_BUFSIZE) {
-        MBED_WARNING(MBED_MAKE_ERROR(MBED_MODULE_DRIVER, MBED_ERROR_CODE_ENOBUFS), \
-                     "ESP8266::_packet_handler(): \"esp8266.socket-bufsize\"-limit exceeded, packet dropped");
+        tr_debug("\"esp8266.socket-bufsize\"-limit exceeded, packet dropped");
```

Comment:
Why these have been changed from `MBED_WARNING` to `tr_debug()`

Reply:
Based on @kjbracey-arm's comment about usage of MBED_WARNING.